src/wallet/spend.h

```diff
@@ -125,6 +125,26 @@ std::optional<SelectionResult> AttemptSelection(const CWallet& wallet, const CAm
 std::optional<SelectionResult> ChooseSelectionResult(const CWallet& wallet, const CAmount& nTargetValue, const CoinEligibilityFilter& eligibility_filter, const std::vector<COutput>& available_coins,
                         const CoinSelectionParams& coin_selection_params);
 
+// User manually selected inputs that must be part of the transaction
+struct SelectedInputs
```

Comment:
why not call this `PreSelectedInputs` or `ManualInputs`? `SelectedInputs` doesn't really tell me who selected them and could easily be confused with a `SelectionResult`

Reply:
sure